docs/content/guide/README.md

```diff
@@ -11,13 +11,17 @@ DXOS applications work offline, share state instantly when online, and leave end
 Composer is a collaborative productivity app where developers can build and organize knowledge, extend with custom data and UI, and run private AI against their knowledge locally.
 
 :::note
-DXOS is under development and will continue to change frequently.<br/>Your feedback is most welcome on [GitHub](https://github.com/dxos/dxos/issues) and [Discord](https://dxos.org/discord). <br/>See our [Contribution Guide](https://github.com/dxos/dxos/blob/main/CONTRIBUTING.md) about contributing code.
+The DXOS SDK is under active development but the focus of the core team is on supporting the use cases in Composer for the time being.
```

Comment:
@Raymmar

Reply:
I would try something like: 

"The DXOS SDK is under active development, however our current focus is building out features and functionality that support development of Composer. To this end we will support developers using the SDK, but plan to prioritize requests and suggestions that will help us push Composer forward. 

If you are interested in working with the SDK or contributing to Composer please share feedback on [GitHub](https://github.com/dxos/dxos/issues) or in [Discord](https://dxos.org/discord)."

I think we need to be a bit more clear about our intention here and push it to be more of a call to action. Feel free to tweak it, but this is how I would position it. 

Also, As I read the rest of this, I might want to take a pass at some of this more broadly. Let's connect this week or early next week to review some of the stuff I have been writing for Composer and see if you like the tone and direction it is headed.